### PR TITLE
(v3): docs mobile navigation breadcrumbs

### PR DESCRIPTION
**Pull request by isBatak**
https://github.com/user-attachments/assets/a9c437b9-c97c-4870-bf7e-1c947e909a03

### CODE REVIEW

File: apps/www/app/docs/sidebar.tsx [hunk 1]
```diff
@@ -105,6 +108,35 @@ const MobileMenuButton = chakra("button", {
   },
 })
 
+export const MobileMenuBreadcrumbs = () => {
```
Comment (segunadebayo) [inline apps/www/app/docs/sidebar.tsx:111]: Can we use the breadcrumb component for this?
https://github.com/chakra-ui/chakra-ui/blob/743421bba95ab7ff2dd7c751976a170383c7360a/apps/compositions/src/examples/breadcrumb-basic.tsx

Comment (isBatak) [reply apps/www/app/docs/sidebar.tsx:111]: Done!

File: apps/www/app/docs/sidebar.tsx [hunk 2]
```diff
@@ -21,6 +23,7 @@ import {
 import { usePathname } from "next/navigation"
 import { useEffect, useRef, useState } from "react"
 import { AiOutlineClose, AiOutlineMenu } from "react-icons/ai"
+import { ChevronRightIcon } from "../../../../packages/react/src/components/icons"
```
Comment (segunadebayo) [inline apps/www/app/docs/sidebar.tsx:26]: Can we use icons from `react-icons/ai` here

Comment (isBatak) [reply apps/www/app/docs/sidebar.tsx]: done

**Pull request merged**